cfn/handler/errs.go

```diff
@@ -0,0 +1,59 @@
+package handler
```

Comment:
If we use the constants in that package, it will cause cascading changes in the runtime. Also, by added the errorCodes to the handler package, the user can return a errorCode by using `handler.theError` instead of `cfn.theError` .  It follows the same pattern as returning `handler.OperationStatus`

Reply:
Oh well, it's a two-way door anyway ;)